PyInstaller/utils/hooks/qt.py

```diff
@@ -175,19 +189,19 @@ def qt_menu_nib_dir(namespace):
     """
     Return path to Qt resource dir qt_menu.nib on OSX only.
 
-    :param namespace: Import namespace, i.e., PyQt5 or PySide2
+    :param namespace: Import namespace, i.e., PyQt5 or PySide2/6
 
     :return: Directory containing qt_menu.nib for specified namespace
     """
-    if namespace not in ['PyQt5', 'PySide2']:
+    if namespace not in Qt_Libraries:
         raise Exception('Invalid namespace: {0}'.format(namespace))
     menu_dir = None
 
     path = exec_statement("""
     from {0}.QtCore import QLibraryInfo
-    path = QLibraryInfo.location(QLibraryInfo.LibrariesPath)
+    path = QLibraryInfo.location(QLibraryInfo.LibraryPath)
     print(path)
-    """.format(namespace))
+    """.format(namespace)
```

Comment:
`QLibraryInfo.location(QLibraryInfo.LibraryPath)` does not work under either PySide version, actually. Under `PySide2` it is unavailable (`AttributeError: type object 'PySide2.QtCore.QLibraryInfo' has no attribute 'LibraryPath'`) and under `PySide6` it is callable. 

My suggestion was to simply revert your changes, and use `QLibraryInfo.location(QLibraryInfo.LibrariesPath)` as in original code.

Also, you're now missing a trailing bracket from the `exec_statement(` call.